Source/Microsoft.Teams.Apps.FAQPlusPlus/Bots/FaqPlusPlusBot.cs

```diff
@@ -788,10 +788,10 @@ private static Task<TaskModuleResponse> GetTaskModuleResponseAsync(Attachment qu
                     break;
 
                 case Constants.ShareFeedback:
-                    if (string.IsNullOrEmpty(await this.configurationProvider.GetSavedEntityDetailAsync(ConfigurationEntityTypes.FeedbackTeamId).ConfigureAwait(false)))
+                    if (string.IsNullOrEmpty(await this.configurationProvider.GetSavedEntityDetailAsync(ConfigurationEntityTypes.FeedbackTeamId)) == true)
```

Comment:
I don't think you need to have the conditional here because the method of 
```cs
string.IsNullOrEmpty(someStringVariable)
```
produces a boolean value